easybuild/easyconfigs/n/NiBabel/NiBabel-2.3.0-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'NiBabel'
+version = '2.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://nipy.github.io/nibabel'
+description = """NiBabel provides read/write access to some common medical and neuroimaging file formats,
+ including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT
+ as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and
+ annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Pillow', '5.3.0', versionsuffix),
```

Comment:
Since we already have Pillow/5.0.0 and 4.3.0 in use for 2017b toolchains, we should avoid adding even more versions of Pillow to 2017b toolchains.

@boegel any suggestions on which version to pick?
scikit-image uses 4.3.0, basemap and CNVkit uses 5.0.0

Reply:
My personal pick would be `Pillow/5.0.0` since `Pillow/4.3.0` was used for the 2016b toolchain too. And since it built with the `foss`/`intel` toolchains there is no risk of conflict with the `2018b` (hopefully).

Reply:
Makes sense to avoid introducing additional `Pillow` versions, since that only makes it more likely to run into conflicts...

@murfreesboro Please change to `5.0.0` and drop the `5.3.0` easyconfig?

Reply:
OK, drop the 5.3.0 eb file and use the 5.0.0 instead now.